src/geometry/Shape.ts

```diff
@@ -287,6 +304,14 @@ export default abstract class Shape extends THREE.Group {
     if (strokeDashOffset !== undefined) {
       this.stroke.material.dashOffset = strokeDashOffset;
     }
+
+    if (config.includeChildren) {
```

Comment:
Seeing this logic made me realize that what I'm actually looking for is an `includeDescendents` option to include everything beneath the shape this is called on 🤔